_pages/ato/walkthrough.md

```diff
@@ -66,6 +66,7 @@ Once all of the materials are prepared and the system is considered "ready" by t
 There are several ways to ensure that your system remains compliant:
 
 * Set up [static analysis](../../security/static-analysis/) to run on an ongoing basis through one of the hosted [services](../../security/static-analysis/#services).
+* Set up [automated vulnerability scanning](../../security/dynamic-scanning/#automated-scanning).
```

Comment:
Enhancement request: Is setting up automated vulnerability scanning required or optional for ATO at 18F right now? Let's note that here.